### PR TITLE
[tests-only][donot-merge] added test for sync folder creation in default path

### PR DESCRIPTION
**Pull request by sushmita56**
Related issue: #9797

A test scenario added in this PR is not a bug demonstration, it is written for the expected behavior of the AUT. The test is failing currently but it must be passed as the [bug](https://github.com/owncloud/client/issues/9797 ) gets fixed.

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>GUI-tests</strong> https://drone.owncloud.com/owncloud/client/12139/6/1

:boom: The GUI tests failed.

GUI Logs: https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/index.html

<details>

Server Logs: https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/serverlog.log

Screenshots:
- [Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/Progress_indicator_should_not_be_visible_after_unselecting_the_password_protection_checkbox_while_sharing_through_public_link.png)
- [change_collaborator_permissions_of_a_file_&_folder.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/change_collaborator_permissions_of_a_file_&_folder.png)
- [share_a_file_with_many_users.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/share_a_file_with_many_users.png)
- [share_file_and_folder_to_a_group.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/share_file_and_folder_to_a_group.png)
- [sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_creates_a_file_and_a_folder_inside_a_shared_folder.png)
- [sharee_deletes_a_file_and_folder_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_deletes_a_file_and_folder_shared_by_sharer.png)
- [sharee_edits_content_of_files_shared_by_sharer.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_edits_content_of_files_shared_by_sharer.png)
- [sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_edits_shared_files_and_again_try_to_edit_after_write_permission_is_revoked.png)
- [sharee_renames_the_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_renames_the_shared_file_and_folder.png)
- [sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_tries_to_create_a_file_and_a_folder_inside_a_shared_folder_without_write_permission.png)
- [sharee_tries_to_delete_shared_file_and_folder_without_permissions.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_tries_to_delete_shared_file_and_folder_without_permissions.png)
- [sharee_tries_to_edit_content_of_files_shared_without_write_permission.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharee_tries_to_edit_content_of_files_shared_without_write_permission.png)
- [sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharing_a_folder_by_public_link_with_"Uploader"_role_and_check_if_file_can_be_downloaded.png)
- [sharing_of_a_file_by_public_link_and_deleting_the_link.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_and_deleting_the_link.png)
- [sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/sharing_of_a_file_by_public_link_with_password_and_changing_the_password.png)
- [simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_default_expiration_date.png)
- [simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_a_file_by_public_link_with_password_and_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_expiration_date.png)
- [simple_sharing_of_file_and_folder_by_public_link_with_password.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_with_password.png)
- [simple_sharing_of_file_and_folder_by_public_link_without_password.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_file_and_folder_by_public_link_without_password.png)
- [simple_sharing_of_folder_by_public_link_with_different_roles.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/simple_sharing_of_folder_by_public_link_with_different_roles.png)
- [unshare_a_reshared_file.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/unshare_a_reshared_file.png)
- [unshare_a_shared_file_and_folder.png](https://cache.owncloud.com/public/owncloud/client/12139/guiReportUpload/screenshots/unshare_a_shared_file_and_folder.png)

</details>

File: test/gui/shared/steps/steps.py [hunk 1]
```diff
@@ -1485,3 +1493,18 @@ def step(context, folderName):
...
+    default_sync_path = waitForObject(newAccount.CHOOSE_LOCAL_DOWNLOAD_DIRECTORY)[
+        'displayText'
+    ]
+    folder_name = str(default_sync_path)
+    clickButton(waitForObject(newAccount.CONF_SYNC_MANUALLY_RADIO_BUTTON))
+    clickButton(waitForObject(newAccount.NEXT_BUTTON))
+    x = squish.waitFor(
+        lambda: os.path.exists(folder_name), context.userData['maxSyncTimeout'] * 500
+    )
+    test.xcompare(True, x)
```
Comment (saw-jan) [inline test/gui/shared/steps/steps.py:1510]: should avoid using single char as variable name
```suggestion
    exists = squish.waitFor(
        lambda: os.path.exists(folder_name), context.userData['maxSyncTimeout'] * 500
    )
    test.xcompare(True, exists)
```

File: test/gui/tst_syncing/test.feature [hunk 2]
```diff
@@ -70,6 +70,17 @@ Feature: Syncing files
         And the user selects "ownCloud" as a remote destination folder
         Then the sync all checkbox should be checked
 
+    @issue-9797
+    Scenario: Sync all is selected by default expected behavior
```
Comment (Talank) [inline test/gui/tst_syncing/test.feature:74]:
```suggestion
    Scenario: Sync all is selected by default (expected behavior)
```
I think the `expected behavior` is extra data, demonstrating how the `Sync all is selected by default` should act. I I would suggest to put it inside brackets . 

Or even better if you rephrase the scenario to a unique title.

**Comment by sushmita56** [comment]
The main purpose of this PR is just to check the bug fix and there is not any task to do in this PR.

**Comment by sushmita56** [comment]
> The main purpose of this PR is just to check the bug fix and there is no task, for now, to do in this PR.

For this reason keeping this PR closed but still on the sprint backlog.

**Comment by saw-jan** [comment]
This is not a PR to be merged, it is just for a bug demonstration.

**Pull request closed without merging**